### PR TITLE
ompgsql: rewrite documentation

### PR DESCRIPTION
**Pull request by PascalWithopf**
Module documentation was rewritten in the new style.

### CODE REVIEW

File: source/configuration/modules/ompgsql.rst [hunk 1]
```diff
@@ -1,157 +1,191 @@
...
 **Available:**    8.32+
 ================  ==========================================================================
 
-**Description**:
+
+Purpose
+=======
 
 This module provides native support for logging to PostgreSQL databases. It's an alternative (with potentially superior performance) to the more generic
 `omlibdbi <omlibdbi.html>`_ module.
```
Comment (deoren) [inline source/configuration/modules/ompgsql.rst]: Suggestion:

Use the Sphinx `:doc:` cross-reference role here.

```
:doc:`omlibdbi`
```

File: source/configuration/modules/ompgsql.rst [hunk 2]
```diff
@@ -1,157 +1,191 @@
...
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
 
-.. function:: template <word>
+   "word", "none", "yes", "none"
 
-   **Default**: none **Required**: false
+The multi-tenant database name to `INSERT` rows into.
```
Comment (deoren) [inline source/configuration/modules/ompgsql.rst]: [Suggestion](http://www.sphinx-doc.org/en/stable/rest.html#inline-markup):

```diff
-`INSERT`
+``INSERT``
```

File: source/configuration/modules/ompgsql.rst [hunk 3]
```diff
@@ -1,157 +1,191 @@
...
-         db="syslog"
-         template="sql-syslog" 
-         queue.size="10000" queue.type="linkedList"
-         queue.workerthreads="5"
-         queue.workerthreadMinimumMessages="500"
-         queue.timeoutWorkerthreadShutdown="1000"
-         queue.timeoutEnqueue="10000")
+   "word", "none", "no", "none"
 
+The template name to use to `INSERT` rows into the database with. Valid SQL
```
Comment (deoren) [inline source/configuration/modules/ompgsql.rst]: [Suggestion](http://www.sphinx-doc.org/en/stable/rest.html#inline-markup):

```diff
-`INSERT`
+``INSERT``
```

**Comment by PascalWithopf** [comment]
@deoren changes are done

**Comment by deoren** [comment]
@PascalWithopf Thanks.

**Pull request merged**